src/protocols/secure_channel/Constants.h

```diff
@@ -71,6 +71,8 @@ enum class MsgType : uint8_t
     CASE_Sigma2Resume = 0x33,
 
     StatusReport = 0x40,
+
+    ICD_CHECKIN = 0X50,
```

Comment:
```suggestion
    ICD_CheckIn = 0X50,
```
Just to keep the syntaxe the same as the others (Protocol_message)